rfd/0187/README.md

```diff
@@ -0,0 +1,141 @@
+- Some licenses may require specific attribution or have restrictions on derivative works
+
+**Intellectual Property:**
+- AI models trained on copyrighted code could potentially reproduce similar patterns
+- Risk of inadvertent copyright infringement through AI suggestions
+- Patent implications for AI-generated implementations
+
+**Liability:**
+- Contributors remain responsible for AI-generated code that causes issues
+- Standard warranty and indemnification considerations apply
+- Insurance implications should be considered for AI-assisted development
+
+**Compliance:**
+- Some industries or contracts may prohibit or restrict AI-generated code
+- Export control considerations for AI-generated cryptographic implementations
+- Data sovereignty issues if AI tools process sensitive information
+
+**Recommendation:** Consult with legal counsel when in doubt about AI-generated content, particularly for security-critical components or when working under specific contractual obligations.
+
+### Unacceptable Use
+
+**Security/Safety:**
+- Generating cryptographic implementations, key generation, or security-critical code
+- Code that handles authentication, authorization, or access control
+- Network security protocols or firewall rules
+
+**Legal/Compliance:**
+- Code that could infringe on patents or copyrights
+- Generating entire functions/modules that might be substantially similar to existing copyrighted code
+- Code that processes personal data without proper privacy considerations
+
+**Quality/Reliability:**
+- Production database migration scripts or schema changes
+- Code that directly manipulates critical system state without human review
+- Performance-critical code paths without benchmarking/validation
+
+**Architectural:**
+- Core system architecture decisions (API design, data models, etc.)
+- Code that establishes new dependencies or external integrations
+- Changes to build systems, CI/CD pipelines, or deployment processes
+
+**Context-Specific:**
+- Code that requires deep domain knowledge of Triton's specific architecture
+- Modifications to existing complex algorithms without understanding the original intent
+- Code that interfaces with hardware or low-level system components
+
+### Examples
+
+- Commit message of a human crediting Claude: https://github.com/TritonDataCenter/triton-moirai/pull/4/commits/36ee26c3700cc74aa730489abc1779b0de724c01
+- Commit from Claude: https://github.com/TritonDataCenter/triton-moirai/pull/4/commits/87d42eb21886ae2ad9877621bd1144a975ae40db
+- Commit from Claude with Claude as author: https://github.com/TritonDataCenter/rfd/pull/159/commits/a75057c51b8515eb153105ad133b7df7dcee5b3e
+- Squashed commits from Claude with model details added manually: https://github.com/TritonDataCenter/triton-moirai/pull/4/commits/dd12c0a18b2334fc56926d29659c7923d3c68c63
+
+Example of a commit message for a squashed commit where Claude was used:
+
```

Comment:
One of these for each of the above examples, please?  You've 1/4 here (the squashed-commit), but the other three might benefit from seeing what `git log -1 $COMMIT` shows.

Reply:
Added. Please take a look.